testing/test_service_assertions.py

```diff
@@ -0,0 +1,231 @@
+import os
+from abc import ABC, abstractmethod
+
+import numpy as np
+import pytest
+from sklearn.metrics import mean_squared_error
+
+from neuraxle.base import Identity, ExecutionContext, ForceHandleMixin, StepWithContext
+from neuraxle.data_container import DataContainer
+from neuraxle.metaopt.auto_ml import InMemoryHyperparamsRepository, AutoML, RandomSearchHyperparameterSelectionStrategy, \
+    ValidationSplitter
+from neuraxle.metaopt.callbacks import ScoringCallback
+from neuraxle.pipeline import Pipeline
+
+
+class BaseService(ABC):
+    @abstractmethod
+    def service_method(self, data):
+        pass
+
+
+class SomeService(BaseService):
+    def service_method(self, data):
+        self.data = data
+
+
+# class SomeStepThatChangesTheRootOfTheExecutionContext(ForceHandleMixin, Identity):
+#     def __init__(self):
+#         Identity.__init__(self)
+#         ForceHandleMixin.__init__(self)
```

Comment:
Usually we delete unused code instead of commenting it. Is this code still useful anyhow?

Reply:
If not, I'd suggest deleting it, and otherwise it'd be good to resolve where to use it in this PR already.

Reply:
deleted